.github/workflows/xforms_builder.yml

```diff
@@ -2,7 +2,7 @@ name: Build xforms image
 
 on:
   workflow_dispatch:
-  push:
+ # push:
     # branches:
     # - master
     # - FS-389
```

Comment:
Could we just delete this file? It will still be in git history so could be restored if it was found to be needed?